Client/Frontend/Home/ReaderPanel.swift

```diff
@@ -442,7 +442,7 @@ class ReadingListPanel: UITableViewController, HomePanel, SWTableViewCellDelegat
             profile.readingList?.updateRecord(record, unread: false)
             // Reading list items are closest in concept to bookmarks.
             let visitType = VisitType.Bookmark
-            homePanelDelegate?.homePanel(self, didSelectURL: encodedURL, visitType: visitType)
+            homePanelDelegate?.homePanel(self, didSelectURL: encodedURL, visitType: visitType, inNewTab: true)
```

Comment:
I made links from the reader panel open in new tabs. All others open in the current tab.